simulator-archetypes/archetype-ws/src/main/resources/archetype-resources/pom.xml

```diff
@@ -11,13 +11,13 @@
   <packaging>war</packaging>
 
   <properties>
-    <java.version>1.8</java.version>
+    <java.version>11</java.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <citrus.version>2.8.0</citrus.version>
-    <citrus.simulator.version>1.2-SNAPSHOT</citrus.simulator.version>
-    <spring.boot.version>2.0.9.RELEASE</spring.boot.version>
-    <testng.version>6.14.3</testng.version>
+    <citrus.version>3.0.0</citrus.version>
```

Comment:
do we want to use `3.0.0` or update to latest (which is, I believe, `3.1.0`)? that question regards all `citrus.version`s, of course.